documentation/book/assembly-overview.adoc

```diff
@@ -21,4 +21,6 @@ This guide describes how to install and use {ProductLongName}.
 
 include::ref-key-features.adoc[leveloffset=+1]
 
+include::con-considerations-for-data-storage.adoc[leveloffset=+1]
```

Comment:
Is this really the right placement for this chapter? I would probably put it somewhere under 2.3 and 3.1.1.

Reply:
What about we leave it where it is, but link back from sections 2.3 and 3.1.1?

It was placed at the same location as the section in the RHEL doc, which also has a section on supported configurations in the opening chapter. (Maybe we should consider a restructure and the possibility of separating the deployment and operator content into separate guides in the future.)

Reply:
Well, I'm not sure I reviewed it when it was originally added to RHEL. But to place it there does not make sense at all for me. It feels like it is completely out of place there. People have no clue how it works but we start telling them something about block storage, iSCSI, JBOD etc. It is even before the document conventions.

Reply:
Thanks Jakub. Let's decide on a new place for 1.1.0. How about:

- RHEL: After '2.3 Installing AMQ Streams'
- OCP: After '2.3 Kafka cluster'

Reply:
That sounds good to me.